cpp/models/ide_secir/model.h

```diff
@@ -197,6 +214,18 @@ class Model
      */
     void compute_recovered();
 
+    /**
+     * @brief Getter for the global support_max, i.e. the maximum of support_max over all TransitionDistributions.
+     *
+     * This determines how many inital values we need for the flows.
```

Comment:
Here it would make sense to say that we don't necessarily need that many flows but then it is ensured that we have enough initial values (as you did in the check_constraints function). Or I would state it differently and say that this determines how many initial flows we compute.